application/values.yaml

```diff
@@ -169,7 +169,7 @@ deployment:
   # Image of the app container
   image: 
     repository: repository/image-name
-    tag: v1.0.0
+    tag: null
```

Comment:
I personally think an empty string is a nicer default value here, as it is already used elsewhere, is still falsy, and conveys that the expected type is string rather than 'null', which avoids confusion. Moreover, `.Values.deployment.image.tag` could still be passed to template functions expecting strings, like `regexReplaceAll`.

Reply:
Also finds for me. I personally prefer to use null as an undefined value than an empty string.